data/typelibrary/io-1.0.0/typelib/eIO/eIWconfig.fbt

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FBType Name="eIWconfig" Comment="Service Interface Function Block Type">
+	<Identification Standard="61499-2">
+	</Identification>
+	<VersionInfo Version="1.0" Author="Maximilian Scharf" Date="2025-05-05">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="CONF" Type="Event" Comment="Configure Request">
+				<With Var="QI"/>
```

Comment:
I don't think that in your config blocks QI and QO is needed. Please remove them.